src/peft/utils/save_and_load.py

```diff
@@ -38,6 +40,8 @@ def get_peft_model_state_dict(model, state_dict=None, adapter_name="default", un
             The name of the adapter whose state dict should be returned.
         unwrap_compiled (`bool`, *optional*, defaults to `False`):
             Whether to unwrap the model if torch.compile was used.
+        base_model_layers_to_save (`list(str)`, *optional*, defaults to `None`):
+            The list of base model layers to save. For example, ["embed_tokens", "lm_head"]
```

Comment:
Let's add a sentence here on what situations the user should pass this argument.